lib/led-strip.cc

```diff
@@ -22,6 +22,10 @@
 #include "multi-spi.h"
 #include "led-strip.h"
 
+#define DO_CIE1931		FALSE
+
+
+/*
```

Comment:
Don't leave commented out regions in the code. If they should go, remove them.

But ... why do you remove the CIE mapping ?